docs/local-cluster-example.md

```diff
@@ -0,0 +1,82 @@
+# Usage Example on Local Cluster
```

Comment:
We can use the provider I already created on the kubevirtci project? (KUBEVIRT_PROVIDER=k8s-multus-1.11.1)

Then the cluster-sync only needs to deploy the ovs plugin daemonset.

Reply:
Donw